assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -111,6 +128,11 @@ const StoriesInspectorControls = (props) => {
     }
   );
 
+  const archiveLink = `<a target="__blank" href="${archiveURL}">${__(
```

Comment:
Why does this open in a window / tab?

Reply:
Should it open in the same tab as the editor? 
I though it might not be a good thing for the user to open the link in the same tab while they are editing it.

Reply:
New tab should be ok for now